src/services/ConsensusInfoClient.ts

```
@@ -42,30 +42,48 @@ class ConsensusInfoClient {
     this._sdkClient ??= this.createClient();
     return this._sdkClient;
   }
-
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)